### DISCUSSION TITLE
Cesium Ion Search provider not working or disabled?

### DISCUSSION THREAD

**Discussion by nicolasvila** [TerriaJS/terriajs#7230] [Q&A]
Hi,

I have an issue with CesiumION search provider. Here is below an extract from my configuration file. 
I tested the token and did some search using curl command. From Terria no request is outgoing.
Bing Maps search is working perfectly on the other hand. What am I doing wrong?
Thank you.
```
{
  "initializationUrls": ["simple", "start"],
  "parameters": {
    "cesiumIonAccessToken": "XXXXX",
    "bingMapsKey": "XXXXX",
    "searchBarConfig": {},
    "searchProviders": [
      {
        "id": "search-provider/cesium-ion",
        "type": "cesium-ion-search-provider",
        "name": "translate#viewModels.searchLocations",
        "url": "https://api.cesium.com/v1/geocode/search/",
        "flightDurationSeconds": 1.5,
        "minCharacters": 5,
        "isOpen": true
      },
      {
        "id": "search-provider/bing-maps",
        "type": "bing-maps-search-provider",
        "name": "translate#viewModels.searchLocations",
        "url": "https://dev.virtualearth.net/",
        "flightDurationSeconds": 1.5,
        "minCharacters": 5,
        "isOpen": true
      }
    ]
  }
}
```

**Comment by na9da** [comment]
Hi @nicolasvila - I can't seem to reproduce the issue with a config similar to the one you shared, both providers working as expected for me. 

The only thing I can think of is that with `minCharacters=5` you need to input at least 5 characters before the network requests are fired. 

Do you see any console errors or the like?
